### PR TITLE
Form implementation

### PR DESCRIPTION
**Pull request by dgsierra**
Implementing the Form Section
- Added contact area.
- Added icons in the bottom.
- Fix a style errors with h1 tags for the cards section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -176,46 +176,65 @@ <h1 class="skills--title">Language</h1>
...
-
     </section>
-    
 
+</div>
 
+<div><section class="contactme">
+    <h3 id="form--title">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+    <form action="https://formspree.io/f/xeqnbagq" method="POST">
+        <input name="full_name" class="contact--input" id="input--name" type="text" maxlength="30" placeholder="Full Name" required>
```
Comment (wuletawwonte) [inline index.html]: - [x] Please avoid using double hyphens(`--`) in the id of an element. It won't go well with javascript. Instead, you can use camelcase or underscores.

File: style.css [hunk 2]
```diff
@@ -304,3 +306,79 @@ li {
...
+  width: 50vw;
+  color: #fff;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 25px;
+  margin: 20px 15vw 20px 15vw;
+  padding: 15px 10px 15px 10px;
+  border: none;
+  cursor: pointer;
+}
```
Comment (wuletawwonte) [inline style.css]: - [ ] Please implement button interaction styling for this button. You can see the design here

File: style.css [hunk 3]
```diff
@@ -304,3 +306,79 @@ li {
...
+
+.contact--input {
+  border-style: hidden;
+  border-bottom: solid 1px #ccc;
+  background-color: rgb(255, 255, 255);
+  padding-top: 25px;
+  margin-top: 30px;
+  text-indent: 20px;
+  padding-bottom: 20px;
+}
```
Comment (wuletawwonte) [inline style.css]: - [x] Please hide the border that appears when the text field is active. You can use `:focus` on the selector and add `border: none` inside the block.

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi team  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @dgonzalesi 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**